README.md

```diff
@@ -33,6 +33,7 @@ Options:
   -t, --file-type <type>            specify which file types GoFSH should accept as input: json-only (default), xml-only, json-and-xml
   --indent                          output FSH with indented rules using context paths
   --meta-profile <mode>             specify how meta.profile on Instances should be applied to the InstanceOf keyword: only-one (default), first, none
+  -a, --alias-filepath <path-to-alias-fsh-file>   specify an existing FSH file containing aliases to be loaded.
```

Comment:
Thank you for making this change! I was actually referring to the example argument field (`<path-to-alias-fsh-file>`) when I suggested changing the phrase to `<alias-filePath>`. Doing so provides just enough space to allow the description to appear in line with the other arguments in the readme file. I think changing the argument name in other locations in the code was a great idea though. 
```suggestion
  -a, --alias-file <alias-filePath> specify an existing FSH file containing aliases to be loaded.
```